js/state/contentSettings.js

```diff
@@ -172,6 +172,16 @@ const getContentSettingsFromSiteSettings = (appState) => {
       addContentSettings(contentSettings.referer, hostPattern, '*', 'allow')
     }
   }
+
+  // temporarySiteSettings
```

Comment:
more generally, there needs to be some way to scope siteSettings/contentSettings by session since in general these settings should not be shared across sessions.